### PR TITLE
Add bazel requirements

### PR DESCRIPTION
**Pull request by liranr23**
Adding to the readme the necassary dependencies we need to install.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Konveyor Forklift controller.
 
 For the build, the forklift uses [Bazel](https://bazel.build/).
 
+Install the following system components:
+
+- bazel >= 5
+- docker
```
Comment (liranr23) [inline README.md:16]: actually not sure about docker, it is required for CI. but now sure if bazel needs it.

Comment (nyoxi) [reply README.md:16]: It is not necessary for build. It is only needed if you want to run the final container.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Konveyor Forklift controller.
...
+- gcc
+- go
```
Comment (nyoxi) [inline README.md:18]: Are you sure? What for? As far as I can tell bazel pulls in go toolset for the project. Specifically version 1.16.5 that we specify in `WORKSPACE`.

File: README.md (hunk 1 above)
Comment (ahadas) [reply README.md:16]: it shouldn't be needed for CI as well but maybe [some tweaks are required](https://github.com/kubevirt/kubevirtci/blob/main/PODMAN.md)

Comment (liranr23) [reply README.md:16]: using podman-docker i didn't try those tweaks. but i had an error about insecure connection to the localhost, when editing the relevant configuration to allow insecure, it still didn't work. with docker it worked out of the box for me. in any case, this is unrelated to the bazel.

**Review by ahadas — APPROVED** [review]

**Pull request merged**